### DISCUSSION TITLE
LaTeX tufte sidenotes?

### DISCUSSION THREAD

**Discussion by dmca-glasgow** [jgm/pandoc#9599] [Q&A] [answered]
Hi there,

I have some LaTeX documents that make heavy use of sidenotes using the tufte-handout class from the [tufte-latex](https://ctan.org/pkg/tufte-latex) package, for example:

```latex
\documentclass{tufte-handout}

\begin{document}

My content \sidenote{My sidenote}.

\end{document}
```

I've noticed that Pandoc omits the sidenote contents completely in Markdown, JSON AST and HTML.

How could I approach writing a custom solution for these sidenotes, given they are not even in the AST?

Thanks.

**Comment by jgm** [comment, marked as answer]
If you use `-f latex+raw_tex`, do they appear in the AST as RawBlock?
If so, you could intercept these in a filter and use pandoc.read on the contents...

**Reply by dmca-glasgow** [reply]
@jgm, yes the AST is now as follows:

```json
{
  "pandoc-api-version": [
    1,
    23,
    1
  ],
  "meta": {},
  "blocks": [
    {
      "t": "Para",
      "c": [
        {
          "t": "Str",
          "c": "My"
        },
        {
          "t": "Space"
        },
        {
          "t": "Str",
          "c": "content"
        },
        {
          "t": "Space"
        },
        {
          "t": "RawInline",
          "c": [
            "latex",
            "\\sidenote{My sidenote}"
          ]
        },
        {
          "t": "Str",
          "c": "."
        }
      ]
    }
  ]
}
```

and I should be able to work with that, thanks!
